src/electionguard/big_integer.py

```diff
@@ -9,7 +9,9 @@
 
 def _hex_to_int(input: str) -> int:
     """Given a hex string representing bytes, returns an int."""
-    return int(input, 16)
+    valid_bytes = input[1:] if (len(input) % 2 != 0 and input[0] == "0") else input
+    hex_bytes = bytes.fromhex(valid_bytes)
```

Comment:
This will throw a `ValueError` for strings that are invalid hexadecimal giving a meaningful error. The previous line is a safeguard for accidental odd leading zeroes exclusively. It could be argued these are not legitimate hex at all and that could be removed.